app/tests/integration/nimbus/pages/new_experiment.py

```diff
@@ -0,0 +1,70 @@
+from pypom import Page, Region
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support import expected_conditions as EC
+
+
+from nimbus.pages.base import Base
+
+
+class NewExperiment(Base):
+    """New Experiment Page."""
+
+    _public_name_locator = (By.CSS_SELECTOR, "#name")
+    _hypothesis_locator = (By.CSS_SELECTOR, "#hypothesis")
+    _application_select_locator = (By.CSS_SELECTOR, "#application")
+    _cancel_btn_locator = (By.CSS_SELECTOR, ".btn-light")
```

Comment:
Q: Do we need to add a better classname or something to this button?

Reply:
we could but I don't think it's a big deal.